db.go

```diff
@@ -2,14 +2,15 @@ package arcticdb
 
 import (
 	"fmt"
+	"path"
 	"sync"
 	"time"
 
 	"github.com/go-kit/log"
+	"github.com/polarsignals/arcticdb/query/logicalplan"
```

Comment:
the linter wants this to be in a separate block as it's an "internal" import rather than an external dependency